coderd/audit_test.go

```diff
@@ -93,9 +97,25 @@ func TestAuditLogsFilter(t *testing.T) {
 				SearchQuery:    "resource_type:template",
 				ExpectedResult: 1,
 			},
+			{
+				Name:           "FilterByUserEmail",
+				SearchQuery:    "user_email:" + coderdtest.FirstUserParams.Email,
```

Comment:
I'd just do `email` instead of `user_email`!

Reply:
Yes, makes sense.